docs/app/Components/ComponentDoc/ComponentExample.js

```diff
@@ -32,7 +32,7 @@ export default class ComponentExample extends Component {
     );
 
     return (
-      <Grid className='one column' style={{marginBottom: '4em', marginTop: '4em'}}>
+      <Grid className='one column' style={{marginBottom: '4em'}}>
```

Comment:
This style was pushing the first example very far away from the dividing header.  Now the first example is place near the dividing header it is under (shown here in orange):

![image](https://cloud.githubusercontent.com/assets/5067638/10556122/3c4ca848-7430-11e5-8ab8-019df9bc8d3f.png)